src/vm/eeconfig.cpp

```diff
@@ -378,6 +378,8 @@ HRESULT EEConfig::Init()
 
 #if defined(FEATURE_TIERED_COMPILATION)
     fTieredCompilation = false;
+    tieredCompilation_tier1CallCountThreshold = 1;
```

Comment:
I'd suggest leaving the default at 30 until we've got results from a number of scenarios to tune it differently.

Reply:
ah, nevermind then : )